localisation/00_zuwc_united_localisation.csv

```diff
@@ -2920,12 +2920,13 @@ EVTOPTA_WCPRM_600;As water wish.;;Wie Wasser w
 EVTOPTB_WCPRM_600;No! You have completely lost your hearing, old man!;;Nein! Ihr habt vollkommen das Geh�r verloren, Alter!;;;;;;;;;;;x
 EVTDESC_WCPRM_605;The oracle frowns and cries: "I'm the voice of the water, young one. The rivers want to warn you. If you decide to remain deaf and go against the tide, well, it is not in my power to convince you.";;Das Orakel runzelt die Stirn und schreit: �Ich bin die Stimme des Wassers, J�ngling. Die Fl�sse wollen dich warnen. Wenn du dich entscheidest, taub zu bleiben und gegen den Strom zu schwimmen, nun, dann liegt es nicht in meiner Macht, dich zu �berzeugen.�;;;;;;;;;;;x
 EVTOPTA_WCPRM_605;Old fool!;;Alter Narr!;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+pick_titanic_branch;Choose Titanic Way;;Titanischen Weg w�hlen;;;;;;;;;;;x
+pick_titanic_branch_desc;Choose the path to follow in honouring the creators.;;W�hlt Euren Pfad, um die Sch�pfer zu ehren.;;;;;;;;;;;x
+picked_branch_cooldown;Recently Picked Religious Branch;;Erst k�rzlich religi�sen Zweig ausgew�hlt;;;;;;;;;;;x
+EVTDESC_WCREL_6000;The makers created us like cogs to a machine: the titan-forged have sculpted mountains, guided and nurtured their assigned domains, and built vast complexes around Azeroth. Each of us is bound to a cause. It is time, to find my purpose. Makers, guide me.;;Die Sch�pfer kreierten uns wie die Zahnr�der einer Maschine: Die Titanengeschmiedeten haben Berge geformt, die ihnen zugewiesenen Gebiete gelenkt und gepflegt und riesige Komplexe in ganz Azeroth errichtet. Jeder von uns ist an eine Sache gebunden. Es ist an der Zeit, meine Bestimmung zu finden. Sch�pfer, leitet mich.;;;;;;;;;;;x
+EVTOPTA_WCREL_6000;Storms, heed my calling!;;St�rme, h�rt mein Rufen!;;;;;;;;;;;x
```

Comment:
```suggestion
EVTOPTA_WCREL_6000;Storms heed my call!;;St�rme, h�rt mein Rufen!;;;;;;;;;;;x
```
@arithon The word 'call' is deliberate compared to 'calling'. Different context. It means that the speaker is telling the Storms to pay attention to his summons.

Reply:
I meant this as a reference to the orc shaman in the wc3 end clip :D

Reply:
actually, you're right... checked on the song again... it's just "heed my call" must've misheard that before